test/integration/update_patient_info_test.rb

```diff
@@ -90,7 +90,9 @@ class UpdatePatientInfoTest < ActionDispatch::IntegrationTest
       find('#pregnancy_household_size_children').select '3'
       find('#pregnancy_insurance').select 'Other state Medicaid'
       find('#pregnancy_referred_by').select 'Other abortion fund'
-      fill_in 'Special circumstances', with: 'Stuff'
+      #fill_in 'Special circumstances', with: 'Stuff'
```

Comment:
Go ahead and remove these depreciated parts of the test -- we've got it working now!